shotover-proxy/src/transforms/mod.rs

```diff
@@ -512,6 +512,15 @@ impl<'a> Wrapper<'a> {
         }
     }
 
+    pub fn messages_to_high_level_string(&mut self) -> String {
```

Comment:
Can we put `#[cfg(test)]` on it? Make it clear you shouldn't be using this outside of a debugging/test context.

Reply:
good idea.
done!